### PR TITLE
Ds ethiop22 dn

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Palaeography filled, minor changes.

### CODE REVIEW

File: DayrSuryan/DSEthiop22.xml [hunk 1]
```diff
@@ -80,14 +80,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                         <foliation>The manuscript is not foliated.</foliation>
                         <collation>
-                           <note>The text block is a multi-quire large format notebook
```
Comment (thea-m) [inline DayrSuryan/DSEthiop22.xml:83]: As I understood the result of https://github.com/BetaMasaheft/Documentation/issues/1418, we agreed to put this in `<p>` in `<support>`. Can I move it there?

**Comment by thea-m** [comment]
I think this is an old branch (the famous DS Ethiop. 120 is back!). It is not dramatic, but I don't want any complications, so after we have finished the conversation, I will copy the changes to a new branch and make a new PR

**Comment by DenisNosnitsin1970** [comment]
Strange, i see the phantom 120 is back, but I did not touch it - at least I think i did not.

File: DayrSuryan/DSEthiop22.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop22.xml:83]: OK, of course.

**Comment by thea-m** [comment]
Must have to do with the resucitation of the old branch, I don't know either...

**Pull request closed without merging**